File: MyMonitorHub.Web/MyMonitorHub.Common/WebApi/Event.cs
using System;

namespace MyMonitorHub.Common.WebApi
{

    /// <summary>
    /// Summary description for Event
    /// </summary>
    public class Event
    {
        public DateTime Timestamp;
        public string Category;
        public string SubCategory;
        public string ItemName; // must be unique within the categories
        public EventType Status;
        public ItemStyle Style;
        public string StatusDescription;
    }

    public enum Status
    {
        
    }
}

File: MyMonitorHub.JavaAgent/src/main/java/mymonitorhub/agent/model/Event.java
package mymonitorhub.agent.model;

import com.fasterxml.jackson.annotation.JsonFormat;

import java.util.Date;

/**
 * Mirrors MyMonitorHub.Common.WebApi.Event — must serialize identically for the server.
 */
public class Event {
    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSXXX")
    public Date Timestamp;
    public String Category;
    public String SubCategory;
    public String ItemName;
    public EventType Status;
    public ItemStyle Style;
    public String StatusDescription;

    public Event() {
    }
}
